fix: edit new flag-value messages (#468)
salesforcecli/plugin-dev: messages/dev.generate.flag.md

# flagDescriptions.duration

A unit and quantity of time, such as "10 minutes" or "4 days". You can optionally specify the minimum, maximum, or default values to take advantage of the built-in CLI validation rules.

# flagDescriptions.option

> Value is a single choice out of a pre-defined list of string values. You will then be asked for the list of allowed values.

Reviewer: It'll ask you if the user can specify multiple values.  It's only a "single choice" if the user says not that prompt.

So I'd like to remove the word "single" ?

Author: Good call.

> Value is a single choice out of a pre-defined list of string values. You will then be asked for the list of allowed values.

Author: ```suggestion
Value is a choice out of a pre-defined list of string values. You will then be asked for the list of allowed values.
```

# flagDescriptions.integer 

An integer. You can optionally specify the minimum, maximum, or default values to take advantage of the built-in CLI validation rules.

# flagDescriptions.custom

Flag with a custom typescript type. You must add some code after the scaffolding is complete.

# flagDescriptions.salesforceId 

A valid Salesforce record ID. You can specify the 3-character prefix and the length of the ID.

# flagDescriptions.file

> A local file. You can specify whether the file must already exist, or if the command will create it if necessary.

Reviewer: there's nothing in the flag for creating files.  A command can create a file, but that's unrelated to the flag

Reviewer: same comment for `directory` below

Author: Oh! gotcha, thanks.

> A local file. You can specify whether the file must already exist, or if the command will create it if necessary.

Author: ```suggestion
A local file. You can specify whether the file must already exist.
```

# flagDescriptions.directory

A local directory path. You can specify whether the directory must already exist.

# flagDescriptions.orgApiVersion

> A valid Salesforce API version number. The code checks for minimal non-retired values. You can optionally use the standard CLI API version flag ("--api-version").

Author: ```suggestion
A valid Salesforce API version number. The code checks for minimal non-retired values. You can optionally mimic the standard CLI API version flag ("--api-version").
```

# flagDescriptions.requiredOrg

> A required Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org").

Reviewer: can these still mention that they work with a default org if there is one?

> A required Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org").

Author: ```suggestion
A required Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally mimic the standard CLI target org flag ("--target-org").  Flag respects the user's default org if it's set. 
```

> A required Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org").

Author: ```suggestion
A required Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org"). Flag respects the user's default org if it's set. 
```

# flagDescriptions.optionalOrg

> An optional Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org").

Author: ```suggestion
An optional Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org"). Flag respects the user's default org if it's set. 
```

# flagDescriptions.requiredHub

> A required Salesforce Dev Hub org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target Dev Hub flag ("--target-dev-hub").

Author: ```suggestion
A required Salesforce Dev Hub org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target Dev Hub flag ("--target-dev-hub"). Flag respects the user's default Dev Hub org if it's set.
```

# flagDescriptions.optionalHub

An optional Salesforce Dev Hub org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target Dev Hub flag ("--target-dev-hub").

# flagDescriptions.url

A valid URL. The code checks that the input matches the official W3C URL specification. Returns the NodeJS Url class for simplified parsing.

# flagDescriptions.string

Any valid string.

# flagDescriptions.boolean

> Value is either "true" (if the user specifies it when executing the command) or "false" (user doesn't specify it.)

Reviewer: ```suggestion
Value is either true (if the user specifies it when executing the command) or false (user doesn't specify it.)
```

If backticks aren't ok, let's take out the quotes so that it doesn't mean **string**.  They're true/false, not "true"/"false"

Author: ```suggestion
Value is either `true` (if the user specifies it when executing the command) or `false` (user doesn't specify it.)
```
